phpstan-baseline.neon

```diff
@@ -270,6 +270,11 @@ parameters:
 			count: 1
 			path: src/lib/View/SearchViewBuilder.php
 
+		-
+			message: "#^Offset 'content_types' does not exist on string\\|null\\.$#"
```

Comment:
I strongly believe that this should be fixed with usage of `\Symfony\Component\HttpFoundation\InputBag::all` instead of adding things to baseline - those will fail with new symfony, as getting nonscalars with `get` is deprecated.

Reply:
This is the code from 2020, I can fix it if needed in the next PR. I need to have unblocked CI for other tasks, without unblocked CI I can't move them to the QA. We should do the ticket first for these changes. Then all those changes should go to the QA